core/src/commonMain/kotlin/com/ustadmobile/core/domain/password/LoginWithSavedPasswordUseCase.kt

```diff
@@ -0,0 +1,5 @@
+package com.ustadmobile.core.domain.password
+
+interface LoginWithSavedPasswordUseCase {
```

Comment:
We can't make two separate requests to credential manager - one for saved password, then another for a passkey.

We need to have one ```GetCredentialUseCase``` - which could return with either a passkey or a saved username/password. The return result can be a sealed class (which would be either a passkey or a username/password).

Reply:
LoginWithSavedPasswordUseCase is no longer used and should be removed. We should also remove ```LoginWithSavePasswordResult``` which is not used anywhere else.